site/hxapi/theatrebreak/av/theatre.md

```diff
@@ -5,7 +5,7 @@
 # Theatre Break Availability
 
 
-## /theatrebreak/{foo}
+## /theatrebreak/<productCode>
 
 ### Method
 
```

Comment:
I've not reviewed theatre yet as it's so low volume.

But best to follow the naming convention - TheatreCode